UnitySDK/Assets/ML-Agents/Examples/3DBall/Scripts/Ball3DAgent.cs

```diff
@@ -44,11 +44,11 @@ public override void AgentAction(float[] vectorAction)
             Mathf.Abs(ball.transform.position.z - gameObject.transform.position.z) > 3f)
         {
             Done();
-            SetReward(-1f);
+            AddReward(-1f);
```

Comment:
Have the environments been retrained with this?  Do you have any results from the training?

Reply:
I reverted those changes